### PR TITLE
[#52] Javascript Tracking Example

### LINKED ISSUE #52: add additional js tracking documentation
Add documentation and example code for tracking via js. Especially cart tracking on PDP

### PR DESCRIPTION
**Pull request by damrath**
fixes #52

### CODE REVIEW

**Comment by toarm** [comment]
we should remove all information about the tracking events and link FF documentation

**Comment by toarm** [comment]
we should mention the fieldRoles and the importance of the relation fieldRoles <-> tracked data

**Comment by damrath** [comment]
Looks good to me, but someone else has to approve.

File: markdown/en/documentation/tracking-with-js.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+#### id 
+The **id** parameter has to match the field value of the field configured as `trackingProductNumber ` [field role](https://web-components.fact-finder.de/documentation/field-roles)
+
+
+#### masterId
+The **masterId** parameter has to match the field value of the field configured as `masterArticleNumber` [field role](https://web-components.fact-finder.de/documentation/field-roles)
+
+
+#### channel
+The channel parameter can be retrieved by the `<ff-communicatrion>` element like: `var channel =document.querySelector("ff-communication").channel;` or by the
```
Comment (omikron-fp) [inline markdown/en/documentation/tracking-with-js.md]: Typo: `<ff-communicatrion>`
This `var channel` thing could be formatted more nicely.

File: markdown/en/documentation/tracking-with-js.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+#### masterId
+The **masterId** parameter has to match the field value of the field configured as `masterArticleNumber` [field role](https://web-components.fact-finder.de/documentation/field-roles)
+
+
+#### channel
+The channel parameter can be retrieved by the `<ff-communicatrion>` element like: `var channel =document.querySelector("ff-communication").channel;` or by the 
+following js property `factfinder.communication.globalSearchParameter.channel`.
+
+For more explanation please refer to the [FACT-Finder Tracking Documentation](https://doku.fact-finder.de/endoc/latest/fact-finder-integration/tracking-interface-integration)
```
Comment (omikron-fp) [inline markdown/en/documentation/tracking-with-js.md]: Full stop at the end, please.

File: markdown/en/documentation/tracking-with-js.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+
+### Tracking Examples
+You can use the tracking object like this:
+
+```javascript
+//Example click tracking event
+track.click({
+    query: "bagpack",
```
Comment (omikron-fp) [inline markdown/en/documentation/tracking-with-js.md]: Did you mean _backpack_?

File: markdown/en/documentation/tracking-with-js.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    id: "010-0242-0318",
+    price: 95.95,
+    masterId: "9478a3507589b60ff7a6234a901298ec",
+    count: 99,
+    event: "cart"
+});
+```
+
+### Retrieve product information via Web Components
+If you can't access the necessary field using your shopsystem API you can query FACT-Finder for product information.
```
Comment (omikron-fp) [inline markdown/en/documentation/tracking-with-js.md]: Needs a comma: `...shopsystem API, you can...`

File: markdown/en/documentation/tracking-with-js.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+                        sid: sid,
+                        id: id,
+                        price: 95.95,
+                        masterId: masterId,
+                        count: 99,
+                        event: "cart"
+                    });
+                    
+                } else {
+                    console.log("no found");
```
Comment (omikron-fp) [inline markdown/en/documentation/tracking-with-js.md]: "**not** found", perhaps?

**Comment by damrath** [comment]
I'll quickly implemented @omikron-fp 's suggestions

**Pull request merged**